spk/go2rtc/Makefile

```diff
@@ -0,0 +1,37 @@
+SPK_NAME = go2rtc
+SPK_VERS = 1.6.2
+SPK_REV = 1
+SPK_ICON = src/go2rtc.png
+DSM_UI_DIR = app
+
+DEPENDS = cross/go2rtc
+
+UNSUPPORTED_ARCHS = $(PPC_ARCHS)
+
+MAINTAINER = AlexxIT
```

Comment:
MAINTAINER is not the developer of a package. It is either SynoCommunity or probably you, that is maintaining this package for SynoCommunity.